src/middleware/analytics.ts

```diff
@@ -64,13 +72,12 @@ const trackEvent = ({ clientIdentifier, category, action, keyword }) =>
     }
   });
 
-export default async (req, _, next) => {
+export default async (req: Request, next: NextFunction) => {
```

Comment:
this will through an error since Express is expecting at least three arguments for endpoint handlers - please include `_`